### PR TITLE
try installing bzip2

### PR DESCRIPTION
**Pull request by TomAugspurger**
Just testing this out. xref #17696

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Hitting `unable to execute 'x86_64-conda_cos6-linux-gnu-gcc': No such file or directory
` on [circle now](https://circleci.com/gh/pandas-dev/pandas/5089?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

There were some changes to the compiler

**Comment by TomAugspurger** [comment]
Second failure is due to https://github.com/conda/conda/issues/6030

**Comment by jreback** [comment]
maybe fix the version of mini conda installer for now (to most recent before bug)

File: ci/install_travis.sh [hunk 1]
```diff
@@ -148,6 +148,9 @@ if [ -e ${REQ} ]; then
     time bash $REQ || exit 1
 fi
 
+# test commit
+conda install bzip2
```
Comment (jreback) [inline ci/install_travis.sh]: if we need this is should be somewhat selective and should skip tests if it's not installed

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by TomAugspurger** [comment]
> maybe fix the version of mini conda installer for now (to most recent before bug)

Sure, I'll open another PR for that. Will keep testing on this one

File: ci/install_travis.sh (hunk 1 above)
Comment (TomAugspurger) [reply ci/install_travis.sh]: It's an issue with the recipe in the defaults channel. It's being rebuilt.

I'm kind of surprised that pytables doesn't raise an exception on write here. They fall back to zlib with a warning.

Comment (jreback) [reply ci/install_travis.sh]: yep u to want to raise an issue there

**Pull request closed without merging**